x/evm/ante/router.go

```diff
@@ -0,0 +1,45 @@
+package ante
+
+import (
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+type EVMRouterDecorator struct {
+	defaultRoute sdk.AnteHandler
```

Comment:
How about changing to
- defaultAnteHandler
- evmAnteHandler

Reply:
updated